### PR TITLE
Add publish instructions and download-prebuilds npm script

### LINKED ISSUE #557: Add npm script to pull binaries from CI, before npm publish
Using `prebuildify-ci download`. Also, should we document this required step? @ralphtheninja

### PR DESCRIPTION
**Pull request by vweevers**
Closes #557.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +48,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify --napi --strip",
+    "download-prebuilds": "rimraf prebuilds && prebuildify-ci download",
```
Comment (vweevers) [inline package.json]: Hm, if I'm reading [this](https://github.com/mafintosh/prebuildify-ci/blob/9810c562ca0252d82454aa109dee878a23896033/index.js#L66) right, prebuildify already cleans the directory, so `rimraf` should not be necessary.

File: README.md [hunk 2]
```diff
@@ -363,6 +363,17 @@ Alternatively, you can initialize submodules inside the cloned folder:
...
+### Publishing
+
+1. Increment the version: `npm version ..`
+2. Push to GitHub: `git push --follow-tags`
+3. Wait for Travis and AppVeyor builds to complete
+4. Download prebuilds into `./prebuilds`: `npm run download-prebuilds`
+5. Optionally verify loading a prebuild: `rimraf build && npm t`
+6. Optionally verify which files npm will include: `irish-pub`
+7. Add changelog to the GitHub release
+8. Finally: `npm publish`
```
Comment (ralphtheninja) [inline README.md]: Maybe later we could even have a `prepublish` script (which is run during `npm publish`), which:

1. downloads binaries with `prebuildify-ci`
1. verifies with `rimraf build && npm t`

If either fail then `npm publish` fails.

Comment (vweevers) [reply README.md]: Downside of prepublish script is that it also runs on `irish-pub` IIRC

Comment (vweevers) [reply README.md]: Or rather, that's a downside of `irish-pub` ;)

Comment (vweevers) [reply README.md]: => #565

**Pull request merged**